lib/lsp/lsp_custom.rb

```diff
@@ -34,6 +34,30 @@ def from_h!(value)
     end
   end
 
+  # export interface GetPdkResponse {
+  #   files: string[];
+  #   data: string;
+  #   error: string;
```

Comment:
nit:  If it's optional it should be `error?`

Reply:
This is ruby, do you mean to use typescript syntax here?